command/forceleave/forceleave.go

```diff
@@ -47,10 +52,13 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	if c.prune {
+		err = client.Agent().ForceLeavePrune(nodes[0])
+	}
```

Comment:
Should the regular `ForceLeave` invocation be in an else block here so that they don't both run?

Reply:
_I knew there was a reason I should be doing an else statement and couldn't remember why_ Yes, I will fix, thank you. 🤦‍♀